internal/firmware/export_test.go

```diff
@@ -0,0 +1,34 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
A few files in Pebble have this editor cruft, but most don't. IMO we should drop it. Surely editors a smart enough to use the `.go` file extension these days...

Reply:
Fixed.